sydent/validators/common.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from sydent.db.valsession import ThreePidValSessionStore
+from sydent.validators import ValidationSession
+from sydent.util import time_msec
+
+
+logger = logging.getLogger(__name__)
+
+
+def validateSessionWithToken(sydent, sid, clientSecret, token):
```

Comment:
can haz API doc? i'm particularly wondering how this token is used, and why we need a client secret at all, and what a client secret looks like for MSISDNs, and don't want to have to read the code to work it out